src/app/components/entreprise-form/entreprise-form.component.html

```diff
@@ -0,0 +1,285 @@
+            class="block text-sm font-medium text-gray-700"
+          >
+            Description Brève de l'Entreprise
+          </label>
+          <textarea
+            id="shortDescription"
+            pInputTextarea
+            formControlName="shortDescription"
+            class="mt-1 block w-full rounded-md border-gray-300 p-2 shadow-md outline-byte-200"
+          ></textarea>
+        </div>
+
+        <div class="mb-4">
+          <label
+            for="longDescription"
+            class="block text-sm font-medium text-gray-700"
+          >
+            Description Longue de l'Entreprise
+          </label>
+          <textarea
+            id="longDescription"
+            pInputTextarea
+            formControlName="longDescription"
+            class="mt-1 block w-full rounded-md border-gray-300 p-2 shadow-md outline-byte-200"
+          ></textarea>
+        </div>
+
+        <div class="mb-4">
+          <label for="sectors" class="block text-sm font-medium text-gray-700">
+            Secteurs
+          </label>
+          <p-chips
+            id="sectors"
+            formControlName="sectors"
+            separator=","
+            max="10"
+            class="mt-1 block w-full shadow-md outline-byte-200"
+          ></p-chips>
+        </div>
+
+        <div class="mb-4">
+          <label
+            for="technologiesUsed"
+            class="block text-sm font-medium text-gray-700"
+          >
+            Technologies Utilisées
+          </label>
+          <p-chips
+            id="technologiesUsed"
+            formControlName="technologiesUsed"
+            separator=","
+            max="15"
+            class="mt-1 block w-full shadow-md outline-byte-200"
+          ></p-chips>
+        </div>
+
+        <div class="mb-4">
+          <label for="logo" class="block text-sm font-medium text-gray-700">
+            Logo
+          </label>
+          <input
+            id="logo"
+            pInputText
+            formControlName="logo"
+            type="text"
+            class="mt-1 block w-full rounded-md border-gray-700 p-2 shadow-md outline-byte-200"
+          />
+        </div>
+
+        <div class="mt-6 flex items-center justify-end">
+          <button
+            pButton
+            type="submit"
+            label="Send Data"
+            class="inline-flex justify-center rounded-md border border-transparent bg-byte-500 px-4 py-2 text-sm font-medium text-white shadow-sm outline-none hover:bg-green-900 focus:outline-none focus:ring-2 focus:ring-offset-2"
+          ></button>
+        </div>
+      </div>
+    </form>
+  </p-dialog>
+}
+@if (!entrepriseId) {
+  <div
+    class="mx-auto my-12 rounded-xl bg-white p-6 text-slate-900 shadow-lg md:w-2/3"
+  >
+    <div class="mb-4 text-3xl font-bold text-byte-500">Add Entreprise Data</div>
+    <div class="mb-6 text-base text-slate-700">
+      Whether you are a former employee, a current employee, or a company owner,
```

Comment:
maybe add this paragraph in the same pop up or have the form be in the same page and not in a pop up because it seems like there are two steps, clicking on the button to add or update and then click again on "click here" to open the pop up